.github/workflows/workflow-validation.yml

```diff
@@ -0,0 +1,23 @@
+name: Workflow Validation Testing
+
+on:
+  pull_request:
+    paths:
+      - '.github/**.yml'
+    types:
+      - opened
+      - reopened
+      - synchronize
+
+concurrency:
+  group: workflow-validation-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  visual-regression:
```

Comment:
This looks like it was copied from the VRT workflow. Please be careful when copying/pasting.